src/java/io/bazel/rulesscala/jar/JarCreator.java

```diff
@@ -265,8 +266,18 @@ public static void buildJar(String[] args) throws IOException {
       manifestFile = args[1];
       idx = 2;
     }
+
+    String targetLabel = null;
```

Comment:
Probably usage message needs to be updated. Current impl requires flags in specific order ie `-m ... -t ...` maybe need to be more flexible here?

Reply:
I tried to do a minimal change here - it's a copy/paste from bazel: https://github.com/bazelbuild/bazel/blob/master/src/java_tools/buildjar/java/com/google/devtools/build/buildjar/jarhelper/JarCreator.java

I think better solution would be to research and find a way to depend on the original code and extend/override its functionality. But that's probably too big for this PR.

Reply:
Added info to usage string